linera-client/src/client_context.rs

```diff
@@ -412,30 +412,23 @@ where
         // Take the latest committee we know of.
         let admin_chain_id = self.wallet.genesis_admin_chain();
         let query = ChainInfoQuery::new(admin_chain_id).with_committees();
-        let nodes: Vec<_> = if let Some(validators) = validators {
```

Comment:
Isn't this needed for the case where we request a chain from the faucet in a network where the genesis validators have already disappeared?

We should probably sync the admin chain first, then take the latest committee, and then use that one to download the certificates for the assigned chain.